run_all_tests.sh

```
@@ -0,0 +1,357 @@
+CODE_STATES=("pre-ddp" "post-ddp")
+
+EVAL_BASE_DIR="./eval_results"
+MODEL_BASE_DIR="./regression_models"
+RESULTS_CSV="regression_results.csv"
+
+mkdir -p "$EVAL_BASE_DIR" "$MODEL_BASE_DIR"
+
+# ── Helper: activate environment ─────────────────────────────────────────────
+
+activate_env() {
+    source /home/HDCharles/rhdev/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /home/HDCharles/rhdev/bin/activate was not specified as input (see shellcheck -x).

(SC1091)